mantidproject/mantid
# CropToComponent to work for scanning workspaces

**Description of work.**

This PR relaxes some checks in `IndexInfo` forbidding the operation of `CropToComponent` over scanning workspaces, where one detector is mapped to more than one spectra.

**To test:**

This should run without errors.
```
ws = CreateSampleWorkspace(NumBanks=2, NumScanPoints=5)
b1 = CropToComponent(ws, "bank1")
```



Fixes #22728 . 

Does this update require release notes?


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
